### ISSUE TITLE
In android, open file picker Instead of a gallery

### ISSUE THREAD

**Issue by abbasalim** [Chaintech-Network/CMP-image-pick-n-crop#12]
Thanks for this very fine lib, but in android  open file picker Instead of a gallery, that isn't good 
<img width="421" alt="Screenshot 2024-12-21 at 19 30 01" src="https://github.com/user-attachments/assets/7ae963b1-a1e1-4623-871a-cd4f6ab34075" />

**Comment by ctn-mobile-innovation** [comment]
@abbasalim You're right — on some older Android versions, the system may fall back to showing a generic file picker UI instead of the expected photo gallery interface. This behavior can happen because the newer Photo Picker API is only officially supported on Android 13 (API 33) and above. For lower versions, the library uses a compatible fallback via Intent.ACTION_PICK, but the exact UI still depends on the device manufacturer and installed apps.
We're looking into improving the experience for older devices by adding more consistent fallback behavior. Meanwhile, the file picker shown still allows users to select images, though the UI might vary.
